ccdImports/configFiles/CCD_Probate_Backoffice/CaseTypeTab.json

```diff
@@ -16,6 +16,10 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "deceasedTab", "TabLabel": "Deceased", "TabDisplayOrder": 4, "CaseFieldID": "deceasedAddress", "TabFieldDisplayOrder": 10},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "deceasedTab", "TabLabel": "Deceased", "TabDisplayOrder": 4, "CaseFieldID": "solsDeceasedAliasNamesList", "TabFieldDisplayOrder": 10},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "deceasedTab", "TabLabel": "Deceased", "TabDisplayOrder": 4, "CaseFieldID": "deceasedAliasNameList", "TabFieldDisplayOrder": 11},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "deceasedTab", "TabLabel": "Deceased", "TabDisplayOrder": 4, "CaseFieldID": "deceasedDiedEngOrWales", "TabFieldDisplayOrder": 12},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "deceasedTab", "TabLabel": "Deceased", "TabDisplayOrder": 4, "CaseFieldID": "deceasedDeathCertificate", "TabFieldDisplayOrder": 13},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "deceasedTab", "TabLabel": "Deceased", "TabDisplayOrder": 4, "CaseFieldID": "deceasedForeignDeathCertInEnglish", "TabFieldDisplayOrder": 14},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "deceasedTab", "TabLabel": "Deceased", "TabDisplayOrder": 4, "CaseFieldID": "deceasedForeignDeathCertTranslation", "TabFieldDisplayOrder": 15},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "caseDetailsTab", "TabLabel": "Case details", "TabDisplayOrder": 5, "CaseFieldID": "caseType", "TabFieldDisplayOrder": 1},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "caseDetailsTab", "TabLabel": "Case details", "TabDisplayOrder": 5, "CaseFieldID": "paperForm", "TabFieldDisplayOrder": 2},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "Channel": "CaseWorker", "TabID": "caseDetailsTab", "TabLabel": "Case details", "TabDisplayOrder": 5, "CaseFieldID": "boWillMessage", "TabFieldDisplayOrder": 3},
```

Comment:
Just checking - this all relates to Caseworker - is the solicitor/prof user journey not affected?